src/featureFlags/index.js

```diff
@@ -19,13 +19,32 @@
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-export const ACP_ENABLED = "acpEnabled";
-export const acpEnabled = () => false;
+export const NEW_ACP_UI_ENABLED_FOR = [
+  "https://pod.inrupt.com/jacklawson/profile/card#me",
+  "https://pod.inrupt.com/arneh/profile/card#me",
+  "https://pod.inrupt.com/megoth/profile/card#me",
+  "https://megoth-demo1.inrupt.net/profile/card#me",
+  "https://megoth-demo2.inrupt.net/profile/card#me",
+  "https://megoth-demo3.inrupt.net/profile/card#me",
+  "https://pod.inrupt.com/kyra/profile/card#me",
+  "https://pod.inrupt.com/virginiabalseiro/profile/card#me",
+  "https://pod.inrupt.com/efe/profile/card#me",
+  "https://podbrowser.inrupt.net/profile/card#me",
+  "https://podbrowser2.inrupt.net/profile/card#me",
+  "https://pod.inrupt.com/podbrowser/profile/card#me",
+  "https://pod.inrupt.com/podbrowser2/profile/card#me",
+  "https://pod-compat.inrupt.com/podbrowser/profile/card#me",
+  "https://pod-compat.inrupt.com/podbrowser2/profile/card#me",
+];
+export const NEW_ACP_UI_ENABLED = "newAcpUIEnabled";
+export const newAcpUiEnabled = (session) => {
+  if (!session.info.isLoggedIn) {
+    return false;
+  }
 
-export const WAC_ENABLED = "wacEnabled";
-export const wacEnabled = () => true;
+  return NEW_ACP_UI_ENABLED_FOR.includes(session.info.webId);
```

Comment:
Could also be written `return session.info.isLoggedIn && NEW_ACP_UI_ENABLED_FOR.includes(session.info.webId);`